controller/trackerquery_blackbox_test.go

```diff
@@ -294,14 +297,46 @@ func (rest *TestTrackerQueryREST) TestCreateTrackerQueryID() {
 		assert.Equal(t, trackerquery.Data.ID, result.Data.ID)
 	})
 	rest.T().Run("invalid - fail", func(t *testing.T) {
-		tqpayload := newCreateTrackerQueryPayload(fxt.Spaces[0].ID, fxt.Trackers[0].ID)
+		tqpayload := newCreateTrackerQueryPayload(fxt.Spaces[0].ID, fxt.Trackers[0].ID, fxt.WorkItemTypes[0].ID)
 		invalidID := uuid.Nil
 		tqpayload.Data.ID = &invalidID
 		test.CreateTrackerqueryBadRequest(t, svc.Context, svc, trackerQueryCtrl, &tqpayload)
 	})
 }
 
-func newCreateTrackerQueryPayload(spaceID uuid.UUID, trackerID uuid.UUID) app.CreateTrackerqueryPayload {
+func (rest *TestTrackerQueryREST) TestInvalidWITinTrackerQuery() {
+	t := rest.T()
```

Comment:
Remove this line please.

Reply:
And think about calling the receiver for the suite `s` instead of `rest`. That's shorter and more consistent with the rest of the test code.

Reply:
done here 
0524fee

Reply:
You still have places in which you define `t := s.T()` and in the next line you use `s.T().Run` instead of `t.Run`. Please decide what you want there.

Reply:
I remove `t := s.T()` 071fe92